presto-docs/src/main/sphinx/connector/oracle.rst

```diff
@@ -23,47 +20,354 @@ connection properties as appropriate for your setup:
     connection-user=root
     connection-password=secret
 
-Multiple Oracle Servers
+By default, the Oracle connector uses connection pooling for performance
+improvement. The below configuration shows the typical default values. To update
+them, change the properties in the catalog configuration file:
+
+.. code-block:: properties
+
+    oracle.connection-pool.max-size=30
+    oracle.connection-pool.min-size=1
+    oracle.connection-pool.inactive-timeout=20m
+
+To disable connection pooling, update properties to include:
+
+.. code-block:: none
+
+    oracle.connection-pool.enabled=false
+
+.. note::
+
+    Include catalog files on all of your nodes, this requires you to restart.
+
+Multiple Oracle servers
 ^^^^^^^^^^^^^^^^^^^^^^^
 
-You can have as many catalogs as you need, so if you have additional
-Oracle servers, simply add another properties file to ``etc/catalog``
-with a different name (making sure it ends in ``.properties``). For
-example, if you name the property file ``sales.properties``, Presto
-will create a catalog named ``sales`` using the configured connector.
+If you want to connect to multiple Oracle servers, configure another instance of
+the Oracle connector as a separate catalog.
+
+To add another Oracle catalog, create a new properties file. For example, if
+you name the property file ``sales.properties``, Presto creates a catalog named
+sales.
 
 Querying Oracle
 ---------------
 
-The Oracle connector provides a schema for every Oracle *database*.
-You can see the available Oracle databases by running ``SHOW SCHEMAS``::
+The Oracle connector provides a schema for every Oracle database. Use the below
```

Comment:
This last sentence is misleading, since it's not clear what "below queries" refers to. `SHOW SCHEMAS` will always work. The rest are examples, not something you could directly use.

Reply:
I restructured this section, hopefully its more clear.